templates/bootstrap/plugin_name/app/viewsets.py.j2

```diff
@@ -60,7 +60,7 @@ class {{ plugin_camel_short }}ContentViewSet(core.ContentViewSet):
         "Artifacts" need to be popped off and saved indpendently, as they are not actually part
         of the Content model.
         """
-        raise NotImplementedError("FIXME")
```

Comment:
I think this is more explicit

Reply:
After this change, users who initialized a new Pulp plugin will not get an HTTP 500 `Internal Server Error` response. Instead, they will receive an HTTP 501 `Not Implemented` response.

Reply:
you are right, it seems better